server/controllers/work.js

```diff
@@ -96,7 +97,10 @@ export const search = async (ctx, next) => {
     resultsList,
     query,
     pagination,
-    queryString
+    queryString,
+    promos: worksLandingPromos,
+    henryImage: henryImage,
+    redirectedFrom: null // TODO: update with WI path if it's a redirect
```

Comment:
Could we keep this as part of the PRs that we're going to have to do for that?

Reply:
Done.